apps/condo/domains/marketplace/schema/Invoice.test.js

```diff
@@ -604,6 +608,30 @@ describe('Invoice', () => {
                 expect(invoice).toHaveProperty('toPay', expectedSum)
             })
         })
+
+        describe('The time when some status set', () => {
+
+            test('publishedAt', async () => {
+                const [invoice] = await createTestInvoice(adminClient, dummyInvoiceContext)
+                expect(invoice.publishedAt).toBe(null)
+                const [updatedInvoice] = await updateTestInvoice(adminClient, invoice.id, { status: INVOICE_STATUS_PUBLISHED })
+                expect(dayjs(updatedInvoice.publishedAt).isSameOrAfter(dayjs(invoice.createdAt))).toBe(true)
+            })
+
+            test('paidAt', async () => {
+                const [invoice] = await createTestInvoice(adminClient, dummyInvoiceContext)
+                expect(invoice.paidAt).toBe(null)
+                const [updatedInvoice] = await updateTestInvoice(adminClient, invoice.id, { status: INVOICE_STATUS_PAID })
+                expect(dayjs(updatedInvoice.paidAt).isSameOrAfter(dayjs(invoice.createdAt))).toBe(true)
```

Comment:
You can use `toBeTruthy`
```suggestion
                expect(dayjs(updatedInvoice.paidAt).isSameOrAfter(dayjs(invoice.createdAt))).toBeTruthy()
```

Reply:
I'd like to be sure that type is boolean as well.